generators/java/mixin.cjs

```diff
@@ -65,7 +68,7 @@ module.exports.mixin = parent =>
      * @param {any} config - config to load config from
      * @param {any} into - destination context to use default is context
      */
-    loadJavaConfig(config = this.jhipsterConfig, into = this) {
+    loadJavaConfig(into = this, config = this.jhipsterConfig) {
```

Comment:
`config =defaults({}, this.jhipsterConfig, defaultConfig)`?

Reply:
This is exactly what is happening.
Do you want to drop the parameter?
The parameter is need for workspaces.
```
apps = appConfigs.map(config => this.loadChainConfig({}, config));
```

Reply:
this was just to remove the line 72

Reply:
Unless we save every config to .yo-rc.json, this is not possible.